src/video/openvr/openvr_capi.h

```diff
@@ -46,21 +46,15 @@
 	#endif // OPENVR_API_EXPORTS
 #endif
 
+#if 0 /* SDL handles these, already. */
```

Comment:
This file comes from an external project, so should be minimally touched. If it's causing compile warnings, you can do something like:
`#ifndef USE_SDL`
and then define that before including this header.

Reply:
Done